Automattic/simplenote-macos
# Add beta build lane

This PR adds a lane to build and upload a beta release to AppCenter. The lane takes also care of uploading symbols to Sentry.

### Test
1. Update your `~/.simplenotemacos-default.env` to include the new required params (look at `fastlane/env.example` for the list)
2. `bundle install`
3. Run `bundle exec fastlane build_and_upload_beta`.
4. Verify that the build succeeds. 
5. Verify that the binary has been uploaded to AppCenter.
6. Verify that the symbols have been uploaded to Sentry.

These changes do not require release notes.
